docs/source/private/overview.rst

```diff
@@ -230,16 +230,26 @@ using the name mapping which was created by the ``data_gathering.py`` script.
    wget https://raw.githubusercontent.com/OpenFreeEnergy/IndustryBenchmarks2024/main/industry_benchmarks/utils/rename_exp_data.py
    python rename_exp_data.py --experimental-data my_exp_data.csv -name-mapping-file ligand_name_mapping_PRIVATE.json --output blinded_exp_data.csv
 
+Private datasets info form
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+We also wish to gather additional information about the private datasets:
 
-
-Optionally (if you can't that's completely ok!) we also wish to gather the following:
-
-* Broad assay description (e.g. "Kd from ITC") and free form additional details on the experimental assay.
-* Estimated benchmark difficulty (i.e. "easy", "medium", "hard").
+* Details on system preparation
+* Estimated benchmark difficulty (i.e. "easy", "medium", "hard")
 * Additional details which may impact simulation difficulty, e.g. "likely water sampling issues" or "ions in the binding site".
-* Structure model type, e.g. xray, cryo-em, homology model.
+* Broad assay description (e.g. "Kd from ITC") and additional details on the experimental assay.
 * Compute hardware description, e.g. what type of GPU was used.
-* Any additional information.
+* Any information you have on systematic edge failures
```

Comment:
I think we can remove this as its all included in the template file so they don't have to read it twice.

Reply:
I think if it's ok I would leave them here so they already have an idea of what to expect and prepare without first needing to find and download the file. But if others have strong opinions otherwise, I'm also happy to remove it!